scripts/download_incr_DB.py

```diff
@@ -114,6 +114,13 @@ def GetStateDeltaFromS3(bucketName):
 	GetAllObjectsFromS3(getURL(bucketName), STATEDELTA_DIFF_NAME)
 	ExtractAllGzippedObjects()
 
+def IsDownloadRestartRequired(currTxBlk, latestTxBlk, NUM_FINAL_BLOCK_PER_POW, NUM_DSBLOCK) :
```

Comment:
Can we just use the global variables `NUM_FINAL_BLOCK_PER_POW, NUM_DSBLOCK` in function instead of passing in as param?

Reply:
yeah. correct. changing now.